drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/model/converters/column/impl/BaseColumnConverterUtilities.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.drools.workbench.screens.guided.dtable.client.widget.table.model.converters.column.impl;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.gwtbootstrap3.client.ui.ListBox;
+import org.uberfire.ext.wires.core.grids.client.model.GridCell;
+
+public class BaseColumnConverterUtilities {
+
+    public static void toWidget(final boolean isMultipleSelect,
+                                final GridCell<String> cell,
+                                final ListBox widget) {
+        if (cell == null || cell.getValue() == null || cell.getValue().getValue() == null) {
+            if (widget.getItemCount() > 0) {
+                widget.setSelectedIndex(0);
+            }
+        } else {
+            final String value = cell.getValue().getValue();
+            if (isMultipleSelect) {
+                final List<String> values = Arrays.asList(value.split(","));
```

Comment:
Well, when I looked on this line I got and idea about corner case, when enumeration for string field contains ',' itself. Lets say, I would like to have table cell, that checks if name is contained in list: "Sir, Michael" or "Michael". I was not able to construct such two item list that would appear in the cell list box. Question is, should be I able?

Reply:
@jomarko There are numerous (well, at least one) JIRA about enums containing "," themselves being broken so whatever fix is made for that will touch this code in the future. For the purpose of this PR, use of "," in a value itself is broken and remains broken.

Reply:
@manstis I agree, still my approval holds true.